### PR TITLE
Docs: update the UI language guide.

### LINKED ISSUE #15890: Update the Setting the UI language guide
Update the Setting the UI language guide to match new installation methods: paths, calls, packages, etc.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: update the UI language guide. Closes #15890 

Also closes: cksource/ckeditor5-commercial#5978

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

**Review by arkflpc — APPROVED** [review]

File: docs/getting-started/setup/ui-language.md [hunk 1]
```diff
@@ -41,136 +42,79 @@ See the demo of the editor in Arabic:
...
-* [Zip download](#zip).
+* [npm](#npm)
+* [CDN](#cdn)
 
-Next, you can configure the editor to use the chosen language:
+### npm
+
+After installing the build from npm, languages will be available via `ckeditor5/dist/translations/[lang].js`.
+
+You can load a single language directly to your code by importing. For example, to use Polish, call `'ckeditor5/dist/translations/pl.js'`. Sample editor code will look similar to this one:
```
Comment (filipsobol) [inline docs/getting-started/setup/ui-language.md:56]:
```suggestion
After installing the build from npm, translations can be imported from `ckeditor5/dist/translations/[lang].js` and must be passed to the editor configuration.

For example, to use Polish, import `'ckeditor5/dist/translations/pl.js'` and pass the translation object to the editor configuration:
```

Just a suggestion

File: docs/getting-started/setup/ui-language.md [hunk 2]
```diff
@@ -41,136 +42,79 @@ See the demo of the editor in Arabic:
...
 ### CDN
 
-To use a different language than the default one (English), you need to load the editor together with the preferred language:
+To use different language than the default one (English), you need to load the editor together with the preferred language:
 
 ```html
-<script src="https://cdn.ckeditor.com/ckeditor5/[version.number]/[distribution]/ckeditor.js"></script>
-<script src="https://cdn.ckeditor.com/ckeditor5/[version.number]/[distribution]/translations/[lang].js"></script>
+<script src="https://cdn.ckeditor.com/ckeditor5/dist/ckeditor.js"></script>
+<script src="https://cdn.ckeditor.com/ckeditor5/dist/translations/[lang].js"></script>
```
Comment (filipsobol) [inline docs/getting-started/setup/ui-language.md:84]:
```suggestion
<script src="https://cdn.ckeditor.com/ckeditor5/dist/ckeditor.js"></script>
```

File: docs/getting-started/setup/ui-language.md [hunk 3]
```diff
@@ -41,136 +42,79 @@ See the demo of the editor in Arabic:
...
 ```
```
Comment (filipsobol) [inline docs/getting-started/setup/ui-language.md:85]: These two tags will no longer be required.

Comment (godai78) [reply docs/getting-started/setup/ui-language.md:85]: OK, should I just delete this passage then?

**Pull request merged**